_assets/scss/modules/_thumbnail.scss

```diff
@@ -59,8 +60,14 @@
     display: block;
   }
 
-  .thumb { // image margin bottom
+  /*.thumb { // image margin bottom — replaced by .h4 margin-top
     margin-bottom: 7px;
+  }*/
+
+  .h4 {
```

Comment:
Using the class .h4 is not great here because if we want to change the font to an h3 the css will break. Lets make this .thumbnail-title instead of h4 and add it to the markup.